src/roll20.pyj

```diff
@@ -362,6 +363,12 @@ def rollAttack(request, custom_roll_dice=""):
         damage_types = list(request["damage-types"])
         crit_damages = list(request["critical-damages"])
         crit_damage_types = list(request["critical-damage-types"])
+
+        # Ranger Colossus Slayer Support
+        if damage_types.includes("Colossus Slayer"):

```

Comment:
It might be better to do the indexOf, and check if it's `!= -1` :
```js
    cs_index = damage_types.indexOf("Colossus Slayer")
    if cs_index != -1:
        damages[cs_index] = ROLL20_COLOSSUS_SLAYER_QUERY
```
Also, this being python, it technically should have been `if "Collossus Slayer" in damage_types` but doing it this way is better since I want to move it to JS